src/operators/skipUntil.ts

```diff
@@ -19,8 +19,9 @@ class SkipUntilOperator<T, R> implements Operator<T, R> {
 class SkipUntilSubscriber<T> extends Subscriber<T> {
   private notificationSubscriber: NotificationSubscriber<any> = null;
 
-  constructor(destination: Subscriber<T>, private notifier: Observable<any>) {
-    super(destination);
+  constructor(public destination: Subscriber<T>,
+              private notifier: Observable<any>) {
+    super(null);
```

Comment:
If I do that, how should I write this class to actually unsubscribe when `unsubscribe()` is called?